src/transformers/models/maskformer/feature_extraction_maskformer.py

```diff
@@ -65,9 +65,15 @@ class MaskFormerFeatureExtractor(FeatureExtractionMixin, ImageFeatureExtractionM
         image_std (`int`, *optional*, defaults to `[0.229, 0.224, 0.225]`):
             The sequence of standard deviations for each channel, to be used when normalizing images. Defaults to the
             ImageNet std.
-        ignore_index (`int`, *optional*, default to 255):
-            Value of the index (label) to ignore.
-
+        ignore_index (`int`, *optional*):
+            Value of the index (label) to ignore in the loss function.
```

Comment:
```suggestion
            The index to use to replace background labels. Only relevant in case `reduce_labels` is set to `True`.
```

Reply:
`ignore_index` is always relevant if passed